LICENSE

```diff
@@ -0,0 +1,13 @@
+            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+                    Version 2, December 2004
+
+ Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>
```

Comment:
Shouldn't it be "Copyright (c) 2018 Code Bullet?

Reply:
I just took the default template

Reply:
Yeah, I've seen that a bit later, just forgot to nevermind 😁